metrics-healthchecks/src/main/java/com/codahale/metrics/health/HealthCheck.java

```diff
@@ -73,20 +78,50 @@ public static Result unhealthy(String message, Object... args) {
          * Returns an unhealthy {@link Result} with the given error.
          *
          * @param error an exception thrown during the health check
-         * @return an unhealthy {@link Result} with the given error
+         * @return an unhealthy {@link Result} with the given {@code error}
          */
         public static Result unhealthy(Throwable error) {
-            return new Result(false, error.getMessage(), error);
+            return new ResultBuilder(error).build();
+        }
+
+		/**
+         * Returns an healthy {@link ResultBuilder}
+         *
+         * @return an healthy {@link ResultBuilder}
+         */
+        public static ResultBuilder healthyBuilder() {
+            return new ResultBuilder(true);
+        }
+
+        /**
+         * Returns an unhealthy {@link ResultBuilder}
+         *
+         * @return an unhealthy {@link ResultBuilder}
+         */
+        public static ResultBuilder unHealthyBuilder() {
```

Comment:
I can do that for unhealthy but not for healthy as it has already a method call healthy(). But I agree, I'm not a big fan of this method name so I'm open to suggestion